compiler/rustc_resolve/src/lib.rs

```diff
@@ -67,7 +67,7 @@ use imports::{Import, ImportKind, ImportResolver, NameResolution};
 use late::{HasGenericParams, PathSource, Rib, RibKind::*};
 use macros::{MacroRulesBinding, MacroRulesScope, MacroRulesScopeRef};
 
-type Res = def::Res<NodeId>;
+pub type Res = def::Res<NodeId>;
```

Comment:
This is more like a local import for a commonly used name.
It's better to add a separate one to rustdoc.